restify/index.js

```diff
@@ -37,6 +37,8 @@ const router = new Router();
 navRoutes.forEach(({ base }) => {
   router.add(base, require(`./vulnerabilities/${base.substring(1)}`));
 });
+// misc routes
+router.add('/cookies', require('./vulnerabilities/cookies'));
```

Comment:
Agreed, but since this is framework specific i didn't want to get carried away implementing cookies for each framework. Similar to the unsafe-file-upload routes, these need to be implemented for each framework individually. We should add a follow-up.